recipes/json2html/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
```

Comment:
Please review the example recipe and adhere to the general style shown there:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml